cfgov/unprocessed/apps/paying-for-college/js/views/financial-view.js

```diff
@@ -94,9 +95,20 @@ const financialView = {
     }
   },
 
+  /**
+   * Event handling for input clicks
+   * @param {object} event - the triggering event
+   */
+  _handleInputClick: function( event ) {
+    const target = event.target;
+    if ( target.value === '$0' ) {
+      target.value = '';
+    }
```

Comment:
Doesn't need to be addressed now, but it might make sense to remove the `$` from all fields while they're being edited & then clear out null values.

Reply:
We should discuss this further - I'm not 100% sure I know what you mean. 😀

Reply:
Sure! Just wondering if the dollar sign should generally be removed when editing (so $1200 would become 1200). It's probably not a significant issue to leave it, but it might be more consistent if we're removing `$0`. Then it could be implemented as two steps: remove dollar signs, then remove the remaining contents if they evaluate to 0.

Reply:
UX swoops in: yes, the dollar sign should be removed.

Reply:
I assume also remove commas? Just show a raw number when the user clicks the input? Easy enough to do that, but I don't intend to fix it in this PR. 😄